exporter/otlphttpexporter/config.go

```diff
@@ -12,6 +12,14 @@ import (
 	"go.opentelemetry.io/collector/exporter/exporterhelper"
 )
 
+// EncodingType defines the type for content encoding
+type EncodingType string
```

Comment:
Should we implement `UnmarshalText` to force this type to be either `proto` or `json` when used in a config?